test/powershell/Modules/Microsoft.PowerShell.Utility/Get-Command.Tests.ps1

```diff
@@ -103,3 +103,29 @@ Describe "Get-Command Feature tests" -Tag Feature {
         }
     }
 }
+
+Describe "Get-Command" -Tag CI {
+    Context "-Syntax tests" {
```

Comment:
To make the tests reliable we need to load Microsoft.PowerShell.Management module.
Please add this in `BeforeAll` block

Reply:
I think I've added it in the right place, I couldn't see any other examples of how to do it so figured a simple `Import-Module Microsoft.PowerShell.Management` would be enough.